docs/node/workflows.md

```diff
@@ -65,54 +65,81 @@ There is no official support for querying the state of asynchronously started Wo
 
 ## How to cancel a Workflow Execution
 
-In the Node SDK, Workflows are represented internally by a tree of scopes. The main function runs in the root scope.
-Cancellation propagates from outer scopes to inner ones and is handled by catching `CancelledError`s
-thrown by cancellable operations (see below).
+In the Node SDK, Workflows are represented internally by a tree of **Cancellation Scopes**, each with cancellation behaviors you can specify.
+The Workflow `main` function is the only scope with no parent, running as the "root" scope.
 
-Scopes are created using the [`CancellationScope`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope) constructor or the static helper methods
-[`cancellable`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope#cancellable-1),
-[`nonCancellable`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope#noncancellable),
-and [`withTimeout`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope#withtimeout).
+Scopes are created using the [`CancellationScope`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope) constructor, or one of 3 static helpers:
+
+- [`cancellable(fn)`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope#cancellable-1) : children are automatically cancelled when their containing scope is cancelled.
+  - Equivalent to `new CancellationScope().run(fn)`.
+- [`nonCancellable(fn)`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope#noncancellable): prevents cancellation from propagating to children.
+  - Equivalent to `new CancellationScope({ cancellable: false }).run(fn)`.
+- [`withTimeout(timeoutMs, fn)`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope#withtimeout): If timeout triggers before all activities complete, the Workflow will fail with a `CancelledError`.
```

Comment:
I don't think this is correct. withTimeout should cancel the enclosed fn, but it doesn't really mean that the workflow is canceled. And it is not really related to activities as the fn might not contain any activities.

Reply:
```suggestion
- [`withTimeout(timeoutMs, fn)`](https://nodejs.temporal.io/api/classes/workflow.cancellationscope#withtimeout): If timeout triggers before `fn` resolves the scope will be cancelled, triggering cancellation of enclosed operations, such as activities and timers.
```